### ISSUE TITLE
Elasticsearch installation fails

### ISSUE THREAD

**Issue by joms** [puphpet/puphpet#2128]
It seems getting the installation dependencies for Elasticsearch fails. I tried generating a new config, but no success. Had the same error with previous version of Elasticsearch too.

Log:
```
==> local: Notice: /Stage[main]/Elasticsearch::Package/Exec[create_package_dir_elasticsearch]/returns: executed successfully
==> local: Notice: /Stage[main]/Elasticsearch::Package/File[/opt/elasticsearch/swdl]/seluser: seluser changed 'unconfined_u' to 'system_u'
==> local: Error: wget --no-check-certificate -O /opt/elasticsearch/swdl/elasticsearch-2.1.0.noarch.rpm https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-2.1.0.noarch.rpm 2> /dev/null returned 8 instead of one of [0]
==> local: 
==> local: Error: /Stage[main]/Elasticsearch::Package/Exec[download_package_elasticsearch]/returns: change from notrun to 0 failed: wget --no-check-certificate -O /opt/elasticsearch/swdl/elasticsearch-2.1.0.noarch.rpm https://download.elasticsearch.org/elasticsearch/elasticsearch/elasticsearch-2.1.0.noarch.rpm 2> /dev/null returned 8 instead of one of [0]
==> local: Notice: /Stage[main]/Elasticsearch::Package/Package[elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Package/Package[elasticsearch]: Skipping because of failed dependencies
==> local: Info: Class[Elasticsearch::Package]: Unscheduling all events on Class[Elasticsearch::Package]
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/etc/elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/etc/elasticsearch]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/var/log/elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/var/log/elasticsearch]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/bin]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/bin]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/data]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/data]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/lib]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/lib]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/var/run/elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/var/run/elasticsearch]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/templates_import]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/templates_import]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/scripts]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/usr/share/elasticsearch/scripts]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/etc/init.d/elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/etc/init.d/elasticsearch]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/lib/systemd/system/elasticsearch.service]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/lib/systemd/system/elasticsearch.service]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/Augeas[/etc/sysconfig/elasticsearch]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/Augeas[/etc/sysconfig/elasticsearch]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/etc/elasticsearch/elasticsearch.yml]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/etc/elasticsearch/elasticsearch.yml]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Elasticsearch::Config/File[/etc/elasticsearch/logging.yml]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Elasticsearch::Config/File[/etc/elasticsearch/logging.yml]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/Exec[mkdir_datadir_elasticsearch_es-01]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/Exec[mkdir_datadir_elasticsearch_es-01]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/usr/share/elasticsearch/data/es-01]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/usr/share/elasticsearch/data/es-01]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/Exec[mkdir_configdir_elasticsearch_es-01]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/Exec[mkdir_configdir_elasticsearch_es-01]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/etc/elasticsearch/es-01]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/etc/elasticsearch/es-01]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/etc/elasticsearch/es-01/logging.yml]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/etc/elasticsearch/es-01/logging.yml]: Skipping because of failed dependencies
==> local: Notice: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/etc/elasticsearch/es-01/scripts]: Dependency Exec[download_package_elasticsearch] has failures: true
==> local: Warning: /Stage[main]/Puphpet_elasticsearch/Elasticsearch::Instance[es-01]/File[/etc/elasticsearch/es-01/scripts]: Skipping because of failed dependencies```

**Comment by jtreminio** [comment]
I need your full config.

**Comment by joms** [comment]
Here you go. You need any more? http://pastebin.com/hxun4Tm0

**Comment by jtreminio** [comment]
Fixed by #2222
